packages/e2e-tests/specs/editor/plugins/block-context.test.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import { last } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	activatePlugin,
+	createNewPost,
+	deactivatePlugin,
+	insertBlock,
+	saveDraft,
+} from '@wordpress/e2e-test-utils';
+
+async function openPreviewPage( editorPage ) {
```

Comment:
> Should this be in `e2e-test-utils`? It looks useful outside of this test.

Your instincts are good, since in-fact I [copied this directly from another test](https://github.com/WordPress/gutenberg/blob/5494ff5e528fa7ebe2cb53bf33cf914de3405e60/packages/e2e-tests/specs/editor/various/preview.test.js#L22-L41) 😉 I'll create another follow-up task.

Reply:
> Should this be in `e2e-test-utils`? It looks useful outside of this test.

Issue: #21678